src/components/form/inputs/TextAreaInputNew.tsx

```diff
@@ -104,7 +104,7 @@ export default function TextAreaInputNew({
         }}
         onChange={(e: any) => {
           const newVal = e.target.value;
-          if (name === 'description') {
+          if (name === 'org_description') {
```

Comment:
@saithsab877 this breaks the add org description value if the description is changed to org_description Formik won't be able to pick up the value of the description on submission

```sh

      const newOrg = {
        id: org.id,
        uuid: org.uuid,
        name: body.name || org.name,
        owner_pubkey: org.owner_pubkey,
        img: img || org.img,
        description: body.description || org?.description,
        github: body.github || org?.github,
        website: body.website || org?.website,
        created: org.created,
        updated: org.updated,
        show: body?.show !== undefined ? body.show : org.show,
        bounty_count: org.bounty_count,
        budget: org.budget
      };
```

Reply:
Hi @elraphty,
Got it, thanks for the heads up! I'll change both the field name to 'org_description' and the reference in the body accordingly to keep everything in sync for Formik.